content/ecosystem/integrations/integrations.toml

```diff
@@ -228,9 +239,17 @@ Matrix authentication plugin for Radicale
 """
 maintainer = "etke.cc"
 language = "Python"
-licence = "AGPL-3.0-only"
-repository = "https://gitlab.com/etke.cc/radicale-auth-matrix"
-room = "#source:etke.cc"
+licence = "LGPL-3.0-only"
+repository = "https://github.com/etkecc/radicale-auth-matrix"
+
+[[integrations]]
+name = "go-msc1929"
+language = "Go"
+licence = "LGPL-3.0-only"
+repository = "https://github.com/etkecc/go-msc1929"
+description = """
+Golang client for MSC1929 (/.well-known/matrix/support)
+"""
```

Comment:
This seems to be a library/SDK. If that's correct, can you move it appropriately?